test/quantization/fx/test_quantize_fx.py

```diff
@@ -9049,6 +9085,45 @@ def forward(self, x):
             }
             self.checkGraphModuleNodes(m, expected_node_occurrence=expected_occurrence)
 
+    def test_pixel_unshuffle_module(self) -> None:
+        class MyBias(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.bias = nn.Parameter(torch.randn(64))
+
+        class MyModel(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = nn.Conv2d(8, 8, 1, bias=False)
+		self.unshuffle = nn.PixelUnshuffle(downscale_factor=2)
```

Comment:
an indent issue?
<img width="334" alt="image" src="https://github.com/pytorch/pytorch/assets/1041752/cc4133f2-a70a-4ca1-abfe-2b306e20f987">

Reply:
My bad I used vim for this which I don't normally use and I think it accidentally used hard tabs instead of spaces. I'll fix this and resubmit